clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -166,11 +169,51 @@ public NetworkClient(Selectable selector,
              time,
              discoverBrokerVersions,
              apiVersions,
-             null,
              logContext,
+             Long.MAX_VALUE,
```

Comment:
Seems this constructor is used be Connect Worker which gets passed to ConsumerNetworkClient. So should we not support the consumer client spun in other dependant modules? Do you think the a default value of `Long.MAX_VALUE` for this config makes sense so we can just get the ocnfigured vs default value?

Reply:
Did you mean the NetworkClient in WorkerGroupMember? This PR changes the constructor used in WorkerGroupMember to pass in the configured value. Let me know if I have missed a different one.